docs/audits.md

```diff
@@ -634,7 +634,7 @@ Workflow commands (like `::set-env` and `::add-path`)
 to inject environment variables and therefore obtain code execution).
 
 However, users can explicitly re-enable them by setting the
-`ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable at the workflow,
+`ACTIONS_ALLOW_INSECURE_COMMANDS` environment variable at the workflow,
```

Comment:
We can't change the environment variable anywhere, only the other uses of "unsecure" -- the envvar is GitHub's own misspelling, which we need to detect.

(Sorry, I didn't notice this in your search earlier.)

Reply:
Okay!

Done 🍏

We have 1×unsecure, I've added some "GitHub"-s.

Reply:
Thanks!